testsuite/tests/ui/users_and_roles/test_permissions.py

```diff
@@ -0,0 +1,83 @@
+"""Tests for user permissions in 3scale"""
+
+import pytest
+
+from testsuite.ui.views.admin.audience.billing import BillingView, BillingSettingsView
+from testsuite.ui.views.admin.audience.developer_portal import (
+    DeveloperPortalContentView,
+    CMSNewPageView,
+    CMSNewSectionView,
+    ActiveDocsView,
+)
+from testsuite.ui.views.admin.foundation import AccessDeniedView
+
+
+PERMISSION_DICT = [
+    pytest.param("portal", DeveloperPortalContentView),
+    pytest.param("portal", DeveloperPortalContentView),
+    pytest.param("portal", CMSNewPageView),
+    pytest.param("portal", CMSNewSectionView),
+    pytest.param("finance", BillingView),
+    pytest.param(
+        "finance",
+        BillingSettingsView,
+        marks=[pytest.mark.xfail, pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-10995")],
+    ),
+    pytest.param("plans", ActiveDocsView),
+]
+
+
+def get_views_by_permission(permission_name):
+    """Returns a list of views that has the given permission by permission name"""
+    return [view for perm, view in PERMISSION_DICT if perm == permission_name]
```

Comment:
No usage for this function, and I believe `perm, view in PERMISSION_DICT` would result in too many values to unpack.